src/style.css

```diff
@@ -0,0 +1,101 @@
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 400;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  font-weight: 300;
+  font-style: normal;
+  src: url(./fonts/Avenir-Heavy.ttf);
+}
+
+* {
+  box-sizing: border-box;
+}
+
+body {
+  margin: 8px;
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+}
+
+.form-big {
+  margin: 20px 0;
+}
+
+.form {
+  position: relative;
+}
+
+.big {
+  width: 100%;
+}
+
+.big:before {
+  display: block;
+  position: absolute;
+  content: "";
+  background-image: url(./images/search.svg);
+  background-repeat: no-repeat;
+  height: 19px;
+  width: 19px;
+  top: 25px;
+  left: 26px;
+}
+
+.search-bar-big {
+  height: 70px;
+  padding: 23px 0 23px 62px;
+  font-size: 16px;
+}
+
+.search-bar {
+  font-family: Avenir, Arial, Helvetica, sans-serif;
+  width: 100%;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.search-bar:hover {
+  box-sizing: border-box;
+  background: #fff;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+  border-radius: 4px;
+}
+
+.search-bar:focus {
+  box-sizing: border-box;
+  font-family: Avenir-Heavy, Arial, Helvetica, sans-serif;
+  outline: 0;
+  background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  border: 1px solid #e1e7ed;
+  border-radius: 3px;
+}
+
+.small {
+  width: 100%;
+}
+
+.small:before {
+  display: block;
+  position: absolute;
+  content: "";
+  background-image: url(./images/search.svg);
+  background-repeat: no-repeat;
+  background-size: 11px 11px;
+  height: 11px;
+  width: 11px;
+  top: 15px;
+  left: 13px;
+}
```

Comment:
Do not repeat your self 
![2023-01-23_17h02_31](https://user-images.githubusercontent.com/61904995/214072930-4c7353de-5a2f-429c-bf78-2a4f7b9283a4.png)